plugins/woocommerce-beta-tester/plugin.php

```diff
@@ -16,6 +16,7 @@ function() {
 } );
 
 add_filter( 'woocommerce_admin_get_feature_config', function( $feature_config ) {
+	$feature_config['beta-tester-slotfill-examples'] = false;
```

Comment:
This adds a new feature flag to WCA via beta tester and serves as a way to toggle all examples on/off at once.

In the future, we might want to be able to turn some examples on and others off. In that case, instead of the feature flag we could establish a new tab in WCA Test Helper and provide an individual toggle for each of the example fills.

Reply:
Agreed, I think having a dedicated page for SlotFill testing would help a lot!